src/lib/components/Map/Map.js

```diff
@@ -184,12 +186,27 @@ class Map extends Component {
     const feature = features[0];
     if (!feature) return false;
     const activeLayerObj = this.props.layersObj.find((l) => l.id === feature.layer.id);
+    /**
+     * Todo:
+     * Investigate adjusting zoom to show selected feature
+     */
+    if (feature.layer && feature.layer.id === 'nutrition-sites-live') {
```

Comment:
Is it possible to refactor this and just check for highlight layer property? i.e
`if (feature.layer && feature.layer.id && activeLayerObj['highlight-filter-property'])`

Reply:
Sure this is better than targeting a specific layer by name